core/src/main/java/io/aiven/klaw/controller/UsersTeamsController.java

```diff
@@ -89,7 +90,7 @@ public ResponseEntity<ApiResponse> updateUser(@Valid @RequestBody UserInfoModel
   @PostMapping(
       value = "/updateProfile",
       produces = {MediaType.APPLICATION_JSON_VALUE})
-  public ResponseEntity<ApiResponse> updateProfile(@Valid @RequestBody UserInfoModel updateUserObj)
+  public ResponseEntity<ApiResponse> updateProfile(@Valid @RequestBody ProfileModel updateUserObj)
       throws KlawException {
     return new ResponseEntity<>(
```

Comment:
UsersTeamsControllerService line 159 needs an update to the allUsersAllTenants with the new profile information. At the moment it is updating the DB but not the cache.

Reply:
Nice catch, thanks.